Services/Database/classes/class.ilAuthContainerMDB2.php

```diff
@@ -49,7 +49,7 @@ public function __construct()
 
 		// studip mode: check against submitted md5 password for ilSoapUserAdministration::login()
 		// todo: check whether we should put this to another place
-		if (isset($_POST['password']) && preg_match('/^[a-f0-9]{32,32}$/i', $_POST['password']))
+		if (isset($_POST['password']) && (preg_match('/^[a-f0-9]{32,32}$/i', $_POST['password']) || (strlen($_POST['password']) == 60)))
```

Comment:
I do not recommend to check against 60 characters. I would prefer a substring check against '$2y$' or '$2a$'